lib/GridItem.jsx

```diff
@@ -348,6 +351,7 @@ export default class GridItem extends React.Component<Props, State> {
           (this.props.cancel ? "," + this.props.cancel : "")
         }
         scale={this.props.transformScale}
+        {...this.props.draggableOpts}
```

Comment:
Let's put this at the beginning of the props list so that the other props will override them, in case someon inadvertently sets `handle/cancel` et al.